### PR TITLE
tidy and fix magefile

### PR DESCRIPTION
**Pull request by thereisnogabe**

### CODE REVIEW

**Comment by thereisnogabe** [comment]
@michaelneale what do you think?

**Comment by michaelneale** [comment]
Looks good. Didn’t think of the mobile requirement, nice one.

File: magefile.go [hunk 1]
```diff
@@ -101,28 +93,41 @@ func (w *regexpWriter) Write(p []byte) (int, error) {
...
+		if err := runGoCommand(usr, *cmd); err != nil {
+			logrus.WithError(err).Warnf("Error running command: %s", cmd.String())
+			cmd = exec.Command(Go, "install", goPackage)
+			if err := runGoCommand(usr, *cmd); err != nil {
+				logrus.WithError(err).Fatalf("Error running command: %s", cmd.String())
+				return err
+			}
 		}
-		return cmd.Wait()
+		logrus.Info("Successfully installed govuln")
```
Comment (michaelneale) [inline magefile.go:117]: @decentralgabe this sees to be hard coded - should be `goPackage` right?

**Review by michaelneale — CHANGES_REQUESTED** [review]
I think there was some hard coding

**Comment by thereisnogabe** [comment]
@michaelneale I apologize, I was impatient and merged this into the ssi-sdk project.

**Pull request closed without merging**